tests/test_eval_detection.py

```diff
@@ -420,14 +439,23 @@ def test_remove_duplicate_instances_ground_truth():
     assert metrics.AP.loc["Pedestrian"] == 1.0
 
 
-def test_filter_objs_to_roi():
-    """ Use the map to filter out an object that lies outside the ROI in a parking lot """
+def test_filter_objs_to_roi() -> None:
+    """Use the map to filter out an object that lies outside the ROI in a parking lot."""
     avm = ArgoverseMap()
 
     # should be outside of ROI
     outside_obj = {
-        "center": {"x": -14.102872067388489, "y": 19.466695178746022, "z": 0.11740010190455852},
-        "rotation": {"x": 0.0, "y": 0.0, "z": -0.038991328555453404, "w": 0.9992395490058831},
+        "center": {
```

Comment:
any idea why these got reformatted? did you use `black myfile -l 120`?
do you think it is clearer on 1 line, or multiple lines? I think clearer on one line

Reply:
I'll check on this. I opened: [PR-228](https://github.com/argoai/argoverse-api/pull/228) to hopefully fix formatting related stuff moving forward.

Reply:
Okay, I reran black. All the files should be updated now.